tests/test_date_parsers.py

```
@@ -19,353 +19,335 @@
+        [day_from_ymd(2000,1,2)]
+    assert days_in_strings(['Maybe', 'Today is:[01-02-00]'], order='MDY') == \
+        [day_from_ymd(2000,1,2)]
+    assert days_in_strings(['Is this today?02-01-2000-0', 'Or tomorrow?'],
+                           order='DMY') == [day_from_ymd(2000,1,2)]
+    assert days_in_strings(['Is this today?02-01-2000-0', '2000-01-03!'],
+                           order='DMY') == \
+        [day_from_ymd(2000,1,2), day_from_ymd(2000,1,3)]
+    assert days_in_strings(['Is this today?02-01-2000-0', '2000-01-03!'],
+                           order='DMY', first=True) == day_from_ymd(2000,1,2)
+    assert days_in_strings(['Is this today?02-01-xxx0-0', '2000-xx-03!'],
+                           order='DMY', first=True) == None
```

Ruff (0.15.6)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)